presto-docs/src/main/sphinx/release/release-0.239.rst

```diff
@@ -0,0 +1,51 @@
+=============
+Release 0.239
+=============
+
+General Changes
+_______________
+* Fix  :func:`classification_miss_rate` and :func:`classification_fall_out` functions (:pr:`14740`).
+* Fix NPE in `/v1/thread` end point.
+* Fix an issue where the property ``ignore_stats_calculator_failures`` would not be honored for certain queries that had filters pushed down to the table scan.
+* Fix missing query completion events for queries which fail prior to dispatching.
+* Fix potential infinite loop when the setting ``use_legacy_scheduler`` is set to false.
+* Add aggregation function SET_UNION.
+* Add local disk spilling support for `ORDER BY` syntax.
+* Add local disk spilling support for aggregation functions with `ORDER BY` or `DISTINCT` syntax.
+* Add optimization for cursor projection & filter by extract and compute common subexpressions among all projections & filter first. This optimization can be turned off by session property ``optimize_common_sub_expressions``.
+* Add support for 2 security modes for views. The default `DEFINER` security mode is the same as the previous behavior. Tables referenced in the view are accessed using the permissions of the view owner (the **creator** or.
+* Add support for warning on unfiltered partition keys using `partition-keys-to-warn-on-no-filtering` system property.
```

Comment:
I updated the description with the release note for this PR for https://github.com/prestodb/presto/pull/14800 but I can't seem to open a PR against your repo to update it here.